docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,298 @@
+---
+Author: "Juan Nunez-Iglesias <mailto:jni@fastmail.com>"
+Author: "Andy Sweet <mailto:andrewdsweet@gmail.com>"
+Author: "Kevin Yamauchi <mailto:kevin.yamauchi@gmail.com>"
+Created: '2022-03-23'
+Status: Active
+Type: Process
+---
+
+(nap0)=
+
+# NAP 0 — Purpose and Process
+
+## What is a NAP?
+
+NAP stands for Napari Advancement Proposal. A NAP is a design document
+providing information to the community, or describing a new feature for
+napari or its processes or environment. The NAP should provide a rationale
+for the proposed change as well as a concise technical specification, if
+applicable.
+
+We intend NAPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for documenting
```

Comment:
This actually does mean the broader community, including users. NAPs are announced on all major communications channels used by napari, so the broader community will absolutely have a way to be involved.

Reply:
That makes sense! We should definitely encourage review by non-developer users when this is rolled out (ex: explain the process on the website) I would caution 2 things:
1. github can be intimidating/ not in the usual workflow of a non-coding community member (as we learned in year 1 napari survey) so our non coding users may be reticent to engage 
2. broader community involvement may still not be representative of our goal user community, and thus would be valuable to flag when we may have blind spots for a NAP (in which case we may want to implement an MVP while we explore how to make a certain feature broadly accessible, or counter balance potential bias in other ways).

Again, the NAP sounds like it may be more heavily focused on technical implementation consensus, in which case these user motivation curiosities would  be a pre-step.

Reply:
re github, I'll just note that we do aim to discuss NAPs on the imagesc forum, so that'll cast a slightly broader net. However, you're right that long term, as we have more and more non-coding users, there needs to be a separate process for user feedback. I think that is beyond the scope of NAPs.

napari here is quite unique because none of the things we are based on (PEPs, NEPs, etc) target non-coders *at all*. Since we need NAPs semi-urgently, though, I suggest we discuss this (whether to roll user feedback into the NAP process, or to create a different process for non-coding users) in a future issue.